js/models/stock.js

```diff
@@ -2,11 +2,24 @@
 function Stock(symbol) {
   this.symbol = symbol;
   this.numShares = 0;
+  this.name = name;
+
 
   // your code to search the API and return a Stock object here
 
-}
+  this.getStock = function getStock(){
+    $.ajax({
+      url: "http://dev.markitondemand.com/api/v2/quote/jsonp?symbol=" + symbol,
+      method: "get",
+      dataType: "jsonp"
+    }).done(function(response){
+      name = response.Name,
+      price = response.LastPrice;
```

Comment:
Here you're defining `price` as a global variable. If you're going to have a global variable, declare it at the very top of your entire script, even if it's just to say `var price = null;` or something.